Sming/Arch/Host/Components/hostlib/startup.cpp

```diff
@@ -291,8 +292,8 @@ int main(int argc, char* argv[])
 
 #ifndef DISABLE_NETWORK
 		if(config.enable_network) {
-			lwip_initialised = host_lwip_init(&config.lwip);
-			if(lwip_initialised) {
+			if(host_lwip_init(config.lwip)) {
+				extern void host_wifi_lwip_init_complete();
```

Comment:
@mikee47  Can you move this line to the beginning of the file or a separate header file?